backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraSingleton.java

```diff
@@ -35,13 +35,20 @@ public static void incrementTestsPlayed() {
     }
 
     // Call this method to ensure that cassandra is restarted every MAX_TEST_PLAYED tests
-    public static void restartAfterMaxTestsPlayed() {
+    public static void restartAfterMaxTestsPlayed(Runnable before, Runnable after) {
```

Comment:
What about https://github.com/linagora/james-project/pull/2963#discussion_r355582972?

Reply:
If it's not what the fixup does then I did not understood your proposal.

Can you be more explicit?

Reply:
Runnable doesn't allow to throw, you could take a type that allows checked exception to avoid wrapping lambdas into `Throwing`